serialization/dot/src/test/java/net/automatalib/serialization/dot/DOTDeserializationTest.java

```diff
@@ -58,7 +62,7 @@ public void testRegularDFADeserialization() throws IOException {
                 DOTParsers.dfa().readModel(DOTSerializationUtil.getResource(DOTSerializationUtil.DFA_RESOURCE)).
                         model;
 
-        Assert.assertTrue(Automata.testEquivalence(dfa, parsed, dfa.getInputAlphabet()));
+        checkIsomorphism(dfa, parsed, dfa.getInputAlphabet());
```

Comment:
What does `checkIsomorphism` do differently than `Automata.testEquivalence` (other than being applicable to non-deterministic systems)? Why change tests for code you haven't (or shouldn't have) touched?

Reply:
Not my change, see commit a3a967d

Reply:
Ok, my bad then. I'll see if I can make any sense out of it.